pkg/providers/vmware/mapper/mapper.go

```diff
@@ -427,6 +432,7 @@ func (r *VmwareMapper) MapVM(targetVmName *string, vmSpec *kubevirtv1.VirtualMac
 	nameParts := strings.Split(hostname, ".")
 	vmSpec.Spec.Template.Spec.Hostname = nameParts[0]
 
+	vmSpec.Spec.Template.Spec.Domain.Machine = kubevirtv1.Machine{Type: q35}
```

Comment:
Please have it verified/"documented" by a unit test so that wouldn't get changed accidentally later.